modules/migration-manual-rollback.adoc

```diff
@@ -0,0 +1,90 @@
+// Module included in the following assemblies:
+// migration/migrating-3-4/troubleshooting.adoc
+// migration/migrating-4-4/troubleshooting.adoc
+// migration/migrating-4-4/troubleshooting.adoc
+[id='migration-manual-rollback_{context}']
+= Manual rollback
+
+You can manually roll back a failed migration. Manual rollback is only required if an application has been quiesced, to prevent data corruption in the PV.
+
+This procedure is not required if the application was not quiesced, because the original application is still running on the source cluster.
+
+.Procedure
+
+. On the target cluster, switch to the migrated project:
++
+----
+$ oc project <project>
+----
+
+. Get the deployed resources:
++
+----
+$ oc get all
+----
+
+. Delete each deployed resource type or scale it to `0`:
++
+----
+$ oc delete <resource_type>
+$ oc scale --replicas=0 <resource_type>/<resource_name>
+----
++
+You must ensure that no applications are running on the target cluster and accessing data on the PVC.
+
+. To stop a DaemonSet without deleting it, update the `nodeSelector` in the YAML file:
```

Comment:
I think that this step might need to be a parallel option with the previous step or might need to be presented first. (I'm assuming that the DaemonSets are listed in the output of `oc get all`, and I don't want users to delete itesm that they might want to just stop.)

Reply:
No, that's not a problem. DaemonSets aren't so easy to kill. A DaemonSet would have to be deleted with `kubectl delete <daemonset>`. They won't delete it by accident.